variables.tf

```diff
@@ -95,13 +95,13 @@ variable "aws_default_region" {
 
 variable "backup_user_name" {
   type        = string
-  default     = "mongodb-backup-s3-bucket"
+  default     = "lina-user-db-mongodb-backup-s3-bucket"
```

Comment:
no need to change default values like this, you can jut pass value from outside and override,
when you change/add/remove some variables here please check/update README.md docs, there is also terraform-docs tool which allows to do this,

we will need to have at least two additional variable one for secret name and another for mapping all secret item keys we need to some custom name, so that same secret can store multiple configs for this module multiple instances